topics/Model_Data_Management.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Model and Data Management"
+---
+
+
+The process of travel modeling involves requires complex sets of code and data that should be organized and managed effectively. The needs and approaches vary widely across users and model types, but generally there is a need to: 
+-	Maintain a consistent and archived base of model code, scripts, and other processes,
+-	manage model inputs and settings,
+-	store outputs and results at various scales
+
+Being able to work with a consistent set of code and inputs is critical for an effective and sustainable workflow. Adopting best practices can also help improve the field’s knowledge over time by more easily facilitating strategies such as back casting based on properly archived historical model data. This article discusses motivations and some best practices on the three topics of maintaining model code, inputs, and outputs. 
+
+#### Model Code
+Travel models can range from being a collection of automation scripts to fully-fledged software programs. In the short term, maintaining consistency within the structures of this code base is important for being able to reliably run code when needed. Additionally, consistency is critical when comparing model runs for scenario analysis. Modelers should be confident that any differences between scenarios are due exclusively to input and settings changes and not due to different code bases. In the longer term, older versions of code must be maintained to revisit previous studies. This is particularly important for forecasts that are used for long-range transportation plans or other forecasts that may not be updated for several years. As additional development will undoubtedly occur on the code base, older versions should be maintained to address ongoing planning topics and policies related to that forecast. 
+
+One approach to managing code is housing model code and scripts on GitHub and regularly updating changes. Specific code versions can be archived as an official release, which can be tied to a planning project (e.g., code that was used for a regional transportation plan in a specific year). The releases feature in GitHub creates a snapshot of code in a specific state that can be accessed at any time. Users can specify a version number with a unique description of the code. Modelers may choose the frequency of releases they create, but they should generally at least have a version associated with any major planning project or milestone. Software developers use the following [symantic versioning system](https://semver.org/) which would be a suitable approach for modelers:
```

Comment:
Would be good to talk about version control software (Git) separately from repository implementations (Git, Bitbucket)